flink-connector-pulsar-e2e-tests/src/test/java/org/apache/flink/tests/util/pulsar/common/FlinkContainerWithPulsarEnvironment.java

```diff
@@ -44,8 +37,8 @@ private static Configuration flinkConfiguration() {
         // Increase the jvm metaspace memory to avoid java.lang.OutOfMemoryError: Metaspace
         configuration.set(TaskManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2048));
         configuration.set(TaskManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(512));
-        configuration.set(JobManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2048));
-        configuration.set(JobManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(512));
+        configuration.set(JobManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2560));
+        configuration.set(JobManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(1024));
```

Comment:
Temporarily workaround. Hopefully we locate why JM metaspace OOM. It can be related to both the size of the final JAR ball and how we load and unload the JAR ball.